app/views/episodes/_form_dropdate.html.erb

```diff
@@ -0,0 +1,52 @@
+<% editing = policy(episode).update? && (episode.published_or_released_date.blank? || episode.released_at_changed?) %>
+
```

Comment:
to my understanding this is the partial that houses all of the date/time/zone fields that are displayed/hidden through the "edit pencil" icon.